README.md

```diff
@@ -23,6 +23,10 @@ Alfresco Content Services deployed via docker-compose or Kubernetes contains the
 3. A Postgres DB  
 4. Alfresco Solr6  
 
+## Deployment options
+* [Deploying on AWS with Kubernetes](docs/helm-deploy-aws_cloud.md)
+* [Deploying with Helm on a local host](docs/helm-deployment-localhost.md)
```

Comment:
Is this section describing how to deploy the helm chart on local host with minikube? The name can be confused with the sub section down in README.md called "Minikube" in "Running with Kubernetes".

I think we should split the docker-compose and minikube from README in the same way as "Deploying on AWS with Kubernetes" is done.

Reply:
Agreed. Will continue working on suggested improvements, as discussed.